projectdo

```diff
@@ -34,13 +34,17 @@ has_command() {
 # evaluates the given command while respecting $QUIET and $DRY_RUN.
 execute() {
   if [ $QUIET = false ]; then
-    echo "$1" $TOOL_ARGS
+    if [ -z "$TOOL_ARGS" ]; then
+      printf '%s' "$1"
+    else
+      printf '%s %s' "$1" "$TOOL_ARGS"
```

Comment:
I think this will pass all the arguments along to the tool as one single argument with spaces. We want to actually pass the arguments along to the tool as multiple arguments. I think we need to preserve the `TOOL_ARGS=$@` below and either keep the `echo` line or something equivalent.